wallaby.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config();
```

Comment:
Why is this necessary?

Reply:
The file itself or `require("dotenv").config()`?

Reply:
If you're asking about `dotenv`, I wanted to let Wallaby know about `process.env.TEST_DB_URL`.

Reply:
The require call